projects/go-lib/styles/_typography.scss

```diff
@@ -102,3 +103,14 @@ strong {
     margin: $column-gutter--half 0;
   }
 }
+
+.go-unordered-list {
+  line-height: 1.6;
+  list-style-position: outside;
+  list-style-type: disc;
+  margin: 0 0 2rem 1.1rem;
```

Comment:
Looking at the `go-ordered-list` class it seems like these should share some common styles. Thought's on trying to consolidate these a bit and make some of the styles match (like spacing, etc)?

Reply:
Good point!

Reply:
Well, actually, they work a bit different from each other... @grahamhency

Reply:
@swimtech20 do they? Aight. Just trying to make these as consistent as possible because they are structured the same in the DOM and once these classes are out there for people to use, if we end up changing the names or structure later that will be a breaking change! 🙃

Reply:
@grahamhency can we maybe pair on this tomorrow? Maybe I'm misunderstanding something, but I do see your point...